tests/parm/rrfs_conus13km_hrrr.nml.IN

```diff
@@ -222,8 +223,8 @@
     lwhtr = .true.
     n_var_lndp = @[N_VAR_LNDP]
     nsradar_reset = 3600
-    nst_anl = .false.
-    nstf_name = 2, 0, 0, 0, 0
+    nst_anl = .true.
```

Comment:
@SamuelTrahanNOAA Do we need to keep nst_anal=.true. in this namelist?

Reply:
But we do not use nst_anal option in our FV3_HRRR.xml (our RRFS)

Reply:
My instructions were to put `nst_anl=.true.` and two people confirmed the test was correct. I can rerun with no nst_anl option if you want.

Reply:
This is only used for the two configurations that try to reproduce the RRFS_dev at 13km (one with and one without radar-tten). The suite is FV3_HRRR.

Reply:
If the suite is FV3_HRRR, then we do not need nst_anl in this namelist.

Reply:
I've removed those two lines.